node/12-user/Dockerfile

```diff
@@ -16,14 +16,12 @@ FROM node:12-stretch
 
 USER node
 
-RUN curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
-
-RUN echo 'export PATH="/home/node/.pyenv/bin:$PATH"' >> ~/.profile
-RUN echo 'export PATH="/home/node/.pyenv/shims:$PATH"' >> ~/.profile
-RUN echo 'eval "$(pyenv init -)"' >> ~/.profile
-RUN echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.profile
-
+# Install Python
 ENV PATH="/home/node/.pyenv/bin:/home/node/.pyenv/shims:${PATH}"
-
-RUN pyenv install 3.7.4
-RUN pyenv global 3.7.4
```

Comment:
Ah, I see node 12 is coupled with Python 3.7.4, should they all be running Python 3.7.4 or should this one be on a forward patch?

Reply:
We should check with @busunkim96, we use `python3` in a couple places in our Node.js applications, here:

https://github.com/googleapis/nodejs-firestore/blob/master/.kokoro/release/docs.sh#L32

and, here:

https://github.com/googleapis/nodejs-firestore/blob/master/.kokoro/publish.sh#L22

I believe there were some restrictions on which Python version could run the tooling in question.